### ISSUE TITLE
Has pretrained model?

### ISSUE THREAD

**Issue by jnulzl** [huangyangyu/NoiseFace#3]
@huangyangyu,Hi if has the pretrained model?

**Comment by huangyangyu** [comment]
> @huangyangyu,Hi if has the pretrained model?

L2-softmax model is trained without pretrained model.
ArcFace model is trained with the corresponding L2-softmax model as pretrained model. These two models are both trained with the same noisy dataset.
